### ISSUE TITLE
Feature Request: OPDS-pse support

### ISSUE THREAD

**Issue by sunjam** [jellyfin/jellyfin-plugin-opds#29]
Its goal is to enrich the OPDS feed with information allowing the client to request a specific page of a document without having to download it completely.  This extension was designed primarily for comic books, to allow reading them on connected devices without having to wait for the book to be completely downloaded.  OPDS-PSE adds new attributes and conventions but does not break compatibility. OPDS-PSE feeds are still valid OPDS feeds.

[What is OPDS-pse](https://anansi-project.github.io/docs/opds-pse/intro)
[Github repo tracking current development of OPDS-pse](https://github.com/anansi-project/opds-pse), maintained by developers of multiple Comic Book and Manga selfhosting projects.
[Originally created by the Ubooquity project](https://vaemendis.net/opds-pse/)

**Comment by foux** [comment]
This would be a very welcome improvement!

**Comment by HammyHavoc** [comment]
Yes, this would be very cool to see. It would allow syncing progress with KOReader, and thus progress between devices. Particularly cool to have your ereader sync up with your reading on other devices.

**Comment by Pachinko0** [comment]
Hello! Im very new so im unsure if its already added.

Is it possible with latest versions to sync reading progress? Or is this feature needed to implement it

**Comment by average-peruvian** [comment]
Hello. I second the question: how's the development of this feature going? Is it open for contributions?
